### DISCUSSION TITLE
Custom style active tab

### DISCUSSION THREAD

**Discussion by xeilen** [tusen-ai/naive-ui#3105] [Q&A]
Hi,
We can add custom styles for each tab using `tab-style`
But how to style the active tab?
adding my own class is not working and using already exits class(`n-tabs-tab--active`) also not working.
Thanks!
![image](https://user-images.githubusercontent.com/58168117/173782142-96453293-08bb-4a52-8cf9-0f86e7b9cb35.png)

**Comment by Sepush** [comment]
you can modify these css vars
![image](https://user-images.githubusercontent.com/39197136/174256214-1425b185-c75b-48f6-b29d-09bac3485b8e.png)
